### PR TITLE
Accept ONNX Constants with empty tensors

### PR DESCRIPTION
**Pull request by tomdol**

### CODE REVIEW

File: ngraph/frontend/onnx_import/src/op/constant.cpp [hunk 1]
```diff
@@ -33,8 +34,21 @@ namespace ngraph
...
+                        std::shared_ptr<default_opset::Constant> constant{nullptr};
+                        try
+                        {
+                            constant = std::make_shared<default_opset::Constant>(
+                                type, tensor.get_shape(), tensor.get_data<T>());
+                        }
+                        catch (const ngraph::ngraph_error& exc)
+                        {
+                            NGRAPH_WARN << "Could not create an nGraph Constant for an ONNX "
+                                           "Constant node. Detailed error:\n"
```
Comment (tsocha) [inline ngraph/frontend/onnx_import/src/op/constant.cpp:46]: OK, there was an error, now what?
If there is a warning it should tell what happened(Constant with scalar 0 has been created) otherwise it can be confusing.

File: ngraph/frontend/onnx_import/src/op/resize.cpp [hunk 2]
```diff
@@ -292,6 +294,9 @@ namespace ngraph
                             data, sizes, scales, attrs)};
                     }
 
+                    attrs.shape_calculation_mode =
+                        default_opset::Interpolate::ShapeCalcMode::scales;
```
Comment (tsocha) [inline ngraph/frontend/onnx_import/src/op/resize.cpp:298]: Why these changes are in the PR which should modify a Constant operator?
Is it intentional?

**Review by tsocha — CHANGES_REQUESTED** [review]

**Review by tsocha — APPROVED** [review]

File: ngraph/frontend/onnx_import/src/op/resize.cpp (hunk 2 above)
Comment (tomdol) [reply ngraph/frontend/onnx_import/src/op/resize.cpp:298]: Yes, the whole PR enables a couple of models which are affected by both changes.

File: ngraph/frontend/onnx_import/src/op/constant.cpp (hunk 1 above)
Comment (tomdol) [reply ngraph/frontend/onnx_import/src/op/constant.cpp:46]: It's just to have the same behavior as we have for empty initializers (graph.cpp:93)

**Review by mbencer — APPROVED** [review]

**Review by postrational — APPROVED** [review]
